vss-accessor/src/vss_error.rs

```diff
@@ -0,0 +1,78 @@
+use crate::vss::{ErrorCode, ErrorResponse};
+use prost::bytes::Bytes;
+use prost::{DecodeError, Message};
+use reqwest::StatusCode;
+use std::error::Error;
+use std::fmt::{Display, Formatter};
+
+/// When there is an error while writing to vss-storage, the response contains a relevant error code.
```

Comment:
This is not just for when writing to storage, right?

But in general, prefer to be declarative and succinct when writing the first sentence:

> A mapping from a VSS server error code. Refer to...

Then any other pertinent information can follow.